src/shared/ipvlan-util.h

```diff
@@ -0,0 +1,27 @@
+/* SPDX-License-Identifier: LGPL-2.1+ */
+#pragma once
+
+#include <netinet/in.h>
+#include <linux/if_link.h>
+
```

Comment:
this should include macro.h, so that `_pure_` and `_const_` used below are definitely defined.